client/src/components/graph/util.ts

```diff
@@ -176,7 +172,7 @@ function getSpatialUrl(s3URI: string) {
     return `https://cellxgene.cziscience.com/spatial-deep-zoom/${datasetVersionId}/`;
   }
 
-  return `http://localhost:5005/d/${datasetVersionId}.cxg/static/deep_zoom/`;
+  return `https://cellxgene.dev.single-cell.czi.technology/spatial-deep-zoom/${datasetVersionId}/`;
```

Comment:
use dev S3 bucket for local development too. We just need to fix dev S3 bucket to avoid CORS error